hudi-common/src/main/java/org/apache/hudi/common/table/log/LogReaderUtils.java

```diff
@@ -62,23 +65,23 @@ private static Schema readSchemaFromLogFileInReverse(FileSystem fs, HoodieActive
     return writerSchema;
   }
 
-  public static Schema readLatestSchemaFromLogFiles(String basePath, List<String> deltaFilePaths, Configuration config)
+  public static Schema readLatestSchemaFromLogFiles(String basePath, List<Pair<String, Long>> logFilePathSizePairs, Configuration config)
```

Comment:
should we jsut pass the `LogFile` object as is without having to deal with Pair?